src/lib/core/CHIPConfig.h

```diff
@@ -2439,7 +2439,7 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
  * @brief Defines the maximum number of path objects, limits the number of attributes being read or subscribed at the same time.
  */
 #ifndef CHIP_IM_SERVER_MAX_NUM_PATH_GROUPS
-#define CHIP_IM_SERVER_MAX_NUM_PATH_GROUPS 4
+#define CHIP_IM_SERVER_MAX_NUM_PATH_GROUPS 7
```

Comment:
Why 7 for default?  Since 7 does not looks like a common random size for a pool.

Reply:
would update it soon.